demo/index.md

```diff
@@ -67,7 +67,81 @@ A baseline `<auro-select>` using `<auro-menu>` and `<auro-menuoption>` elements.
 <!-- AURO-GENERATED-CONTENT:END -->
 </auro-accordion>
 
-### Example with auro-icons in options
+## Nested example
```

Comment:
Let's call this something else as I think the name is confusing with the nested menu example. My suggestion would be to just call it `Select inside a dialog`. And then add wording in the description that the example shows using select inside a container that is smaller than the select when the bib is displayed, e.g. auro-dialog.